app/uk/gov/hmrc/helpfrontend/views/Layout.scala.html

```diff
@@ -25,6 +26,7 @@
         hmrcStandardHeader: HmrcStandardHeader,
         hmrcStandardFooter: HmrcStandardFooter,
         languageSelect: LanguageSelect,
+        standardBetaBanner: StandardBetaBanner,
```

Comment:
Do you want to use this phase banner? Or is it being injected by mistake?

Reply:
oops, good catch